repose-aggregator/services/service-client/impl/src/main/java/org/openrepose/services/serviceclient/akka/impl/AkkaServiceClientImpl.java

```diff
@@ -60,29 +61,39 @@ public UntypedActor create() {
 
 
     @Override
-    public ServiceClientResponse get(String hashKey, String uri, Map<String, String> headers) {
+    public ServiceClientResponse get(String hashKey, String uri, Map<String, String> headers) throws TimeoutException {
```

Comment:
This has now been wrapped in a new `AkkServiceClientException`. In the couple of places that are setting the Response status code, the cause is checked to see if it is an `instanceof TimeoutException`. If it is a `TimeoutException`, then the Response status code is set to `GATEWAY_TIMEOUT`(504); else it is set to `INTERNAL_SERVER_ERROR`(500).